### ISSUE TITLE
Add unique drops for zombies

### ISSUE THREAD

**Issue by lauriys** [Lauriysity/Issues#1]
Something along the lines of enchantment books, maybe unique gear?

**Comment by DerEnderKeks** [comment]
Like a Diamond Shovel with Efficiency X?

**Comment by lauriys** [comment]
...i guess that's an option

**Comment by RileyColak** [comment]
Seems like a good idea. Those items could potentially be dropped by rare, more powerful zombies (Giant Zombies???). The items should probably not be combat related to avoid making the items easy to obtain for people that already have one rare item so like, no swords but maybe weak armour with fire protection, feather falling etc. and other tools with high level enchantments.

**Comment by lauriys** [comment]
I'm thinking about an "affix" system along the lines of Diablo, and other hack and slash games, mainly to make the combat more interesting; fighting a wave of 20 mindless zombies after a wave of 20 mindless zombies might get old pretty quickly. When it comes to items, I want to create some special ones also inspired by these games, like PoE's Orb of Transmutation equivalent, that would upgrade Iron items to Diamond, some way to "infuse" items with items like glowstone dust that would gradually increase the stats of the gear, enchantment books that would work simply by dragging them onto items; generally, make it easy and fast to upgrade your gear, leave a lot of room for further improvements (custom enchantments?) but also having to maintain the gear properly (mcMMO Repair skill comes to mind).
Basically, you **can** create the best gear ever, but it's not going to last forever, you should be able to use that gear in order to obtain more items to create even better gear, sell some of the materials or items, you get the idea. Totally not off-topic.
